from src.exceptions import new_exception

class ModelFitError(new_exception.NewException):
    pass
